src/mesh/curved_mesh.jl

```diff
@@ -135,6 +190,46 @@ function bilinear_mapping(x, y, mesh)
 end
 
 
+# In 3D
+# Trilinear mapping from the reference element to the domain described by the faces
+function trilinear_mapping(x, y, z, mesh)
+  @unpack faces = mesh
+
+  x1 = faces[1](-1, -1) # maped from (-1,-1,-1)
```

Comment:
Since we only have to validate the `faces` once, we decided to create a new function that contains all `@assert` lines. We moved it to the constructor.

Reply:
:+1: